File: src/main/java/com/nafys/emperp/entity/Employee.java
package com.nafys.emperp.entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class Employee {
	
	@Id
	@GeneratedValue
	private Long id;
	private String employeeId;
	private String firstName;
	private String lastName;
	private Long mobileNo;
	private String email;
	private Integer active; 
	private String role;
	private Integer approved;
	
	
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	
	public String getEmployeeId() {
		return employeeId;
	}
	public void setEmployeeId(String employeeId) {
		this.employeeId = employeeId;
	}
	public String getFirstName() {
		return firstName;
	}
	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}
	public String getLastName() {
		return lastName;
	}
	public void setLastName(String lastName) {
		this.lastName = lastName;
	}
	public Long getMobileNo() {
		return mobileNo;
	}
	public void setMobileNo(Long mobileNo) {
		this.mobileNo = mobileNo;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public Integer getActive() {
		return active;
	}
	public void setActive(Integer active) {
		this.active = active;
	}
	public String getRole() {
		return role;
	}
	public void setRole(String role) {
		this.role = role;
	}
	
	
	public Integer getApproved() {
		return approved;
	}
	public void setApproved(Integer approved) {
		this.approved = approved;
	}
	@Override
	public String toString() {
		return "Employee [id=" + id + ", employeeId=" + employeeId + ", firstName=" + firstName + ", lastName="
				+ lastName + ", mobileNo=" + mobileNo + ", email=" + email + ", active=" + active + ", role=" + role
				+ ", approved=" + approved + "]";
	}

}

File: src/main/webapp/src/app/shared/employee.ts
export class Employee {
    id:number;
	employeeId:string;
	firstName:string;
	lastName:string;
	mobileNo:number;
	email:string;
	active:number;
	role:string;
    approved:boolean;
    
    constructor(id:number, employeeId:string, firstName:string, lastName:string, mobileNo:number,
        email :string, active :number, role :string, approved:boolean){
        this.id = id;
		this.employeeId = employeeId;
		this.firstName = firstName;
		this.lastName = lastName;
		this.mobileNo = mobileNo;
		this.email = email;
		this.active = active;
		this.role = role;
		this.approved = approved;
    }
}
